core/src/lib/crypto_openssl.cc

```diff
@@ -1517,12 +1516,9 @@ int InitCrypto(void)
   OpenSSL_add_all_algorithms();
 
 #  ifdef HAVE_ENGINE_LOAD_PK11
```

Comment:
This whole #if/#endif block contains no code anymore.  You can remove it completely